src/Compilers/Core/Portable/Symbols/ITupleTypeSymbol.cs

```diff
@@ -14,7 +13,7 @@ namespace Microsoft.CodeAnalysis
     /// </remarks>
     public interface ITupleTypeSymbol : ITypeSymbol
```

Comment:
As I understand the plan was to delete this interface because we are not adding a special type kind for tuple type symbol. If we keep it, there should be a special type kind, special symbol kind and the symbol cannot be a named type symbol.